pkg/config/config.go

```diff
@@ -148,6 +148,8 @@ type OS struct {
 	Password       string            `json:"password,omitempty"`
 	Environment    map[string]string `json:"environment,omitempty"`
 	Labels         map[string]string `json:"labels,omitempty"`
+
+	PersistentStatePaths []string `json:"persistentStatePaths,omitempty"`
```

Comment:
do we consider to add some checks/limitations for those paths

e.g. /tmp should be excluded

and, if a path is invalid/not existing, what will happen ?

In this commit https://github.com/harvester/harvester-installer/pull/402/files#diff-c49da5a63fc83c7c4dda6020b19d2e217bf5a8bb1911c52b5c60d84793c51149,  some checks are done.

Reply:
**/tmp**: I'm not sure if there is such a user scenario, but I tested it and elemental supports persistence of this path
**not existing path**: this is where users typically use this feature, such as `/var/lib/rook`
**invalid path**: I think we do need to add a validator to verify that the user input is a linux path

Reply:
I did a quick look around and didn't find any existing functions to verify linux paths properly, so I decided to create a separate PR

Reply:
@futuretea @w13915984028 Let's add the validation enhancement separately, I will merge this one first. cc @masteryyh PTAL and feedback are still welcome.

Reply:
added and merged https://github.com/harvester/harvester-installer/pull/487